distributed/active_memory_manager.py

```diff
@@ -15,15 +15,13 @@
 import dask
 from dask.utils import parse_timedelta
 
+from distributed import client
```

Comment:
```suggestion
# Needed to avoid Sphinx WARNING: more than one target found for cross-reference 'TaskState' and 'WorkerState'"
# https://github.com/agronholm/sphinx-autodoc-typehints#dealing-with-circular-imports
from distributed import client
```